package.json

```diff
@@ -34,4 +34,4 @@
     "eslint": "^6.8.0",
     "@react-native-community/eslint-config": "^1.1.0"
   }
-}
\ No newline at end of file
+}
```

Comment:
Please remove this newline: it gets automatically removed by the version bump workflow.

Reply:
I was afraid you you say that :) newline has been removed removed